### DISCUSSION TITLE
Version 2.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#162] [General]
Post your issues here for this specific version.

**Comment by Frontloxian** [comment]
I dont have any issues, but i would highly appreciate if you supported original xbox emulation (through xemu). Only reason i want this is cause the original xbox 360 can play xbox games.

**Reply by shazzaam7** [reply]
This project is focused on Xenia and Xenia only.

**Comment by ghost** [comment]
hi there. i found this bug on the 2.0 version that you cant open snipping tool by pressing PrtSc key when the app is in foreground. i tried it with the 1.15 version and it works there, maybe its my windows being buggy but i am not sure. the dev build after the 2.0.0 one does work as intended now that i tested it

**Reply by shazzaam7** [reply]
I think it's because Xenia Manager v2.0.0 is being Run as Administrator due to the way it loads specific game configuration files. I can see that Print Screen doesn't work when Xenia Manager is focused, but, for me CTRL + Print Screen works fine. I'll see what I can do.

**Comment by NuclearGeneral** [comment]
This is not necessarily an issue, but when Xenia Manager is maximized, it also covers the taskbar. Would it be possible for it to not cover up taskbar when maximized? Kinda sucks having to keep pressing windows key to see taskbar to change windows. Just a suggestion. I love the program by far. It helps keep the games organized and well managed per game for emulator settings. Thumbs up!! :)

**Reply by SRC267** [reply]
Try full screen, not windowed mode, that way the bar won't be blocked

**Reply by NuclearGeneral** [reply]
Not Xenia, but Xenia Manager, The Maximize button next to the Only X button in the top right of Xenia Manager, you press, it goes fullscreen, which is the same thing as maxmized, but maximized programs don't usually cover up the taskbar.

**Reply by shazzaam7** [reply]
This is a "placeholder" while I work on the actual "Fullscreen Mode" that supports the controller.

**Comment by A1eNaz** [comment]
Netplay option is missing from the Installer option
![image](https://github.com/user-attachments/assets/43ffaad7-77ab-4ae4-89db-e50db25ad101)

**Reply by shazzaam7** [reply]
I've disabled it until it either gets v4.0.0 or I make my own repository where I make builds for it.
